<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgImportDB</class>
 <widget class="QDialog" name="DlgImportDB">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SkauTan: Import from DB</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Source:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Filename:</string>
       </property>
       <property name="buddy">
        <cstring>leFileName</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leFileName"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnBrowse">
       <property name="text">
        <string>&amp;Browse...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Items to import:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chbManualTag">
     <property name="text">
      <string>Manual tags</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chbLastPlayedDate">
     <property name="text">
      <string>Last played date</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chbPlaybackHistory">
     <property name="text">
      <string>Complete playback history</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chbLocalRating">
     <property name="text">
      <string>Local rating</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chbCommunityRating">
     <property name="text">
      <string>Community rating</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chbSkipStart">
     <property name="text">
      <string>Skip-start</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chbDeletionHistory">
     <property name="text">
      <string>Song deletion history</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chbSongColors">
     <property name="text">
      <string>Song colors</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnImport">
       <property name="text">
        <string>&amp;Import</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leFileName</tabstop>
  <tabstop>btnBrowse</tabstop>
  <tabstop>chbManualTag</tabstop>
  <tabstop>chbLocalRating</tabstop>
  <tabstop>chbCommunityRating</tabstop>
  <tabstop>chbLastPlayedDate</tabstop>
  <tabstop>btnCancel</tabstop>
  <tabstop>btnImport</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
